.github/workflows/python-linting.yml

```diff
@@ -41,4 +41,11 @@ jobs:
       # Run our linting
       - name: Lint code
         run: |
-          ./bin/task lint
\ No newline at end of file
+          ./bin/task lint
+
+      # Testing
+      - name: Test code
+        env:
+          TNT_EX1_OPENWEATHERMAP_API_KEY: ${{ secrets.TNT_EX1_OPENWEATHERMAP_API_KEY}}
```

Comment:
This is not supposed to happen :exploding_head: you ought to mock the environment variable away. Unit test means to test a standalone unit :kissing_heart: